tests/test-timeout.js

```diff
@@ -65,6 +65,20 @@ tape('should set connect to false', function(t) {
   })
 })
 
+tape('should set host and port', function(t) {
+  var shouldTimeout = {
+    url: s.url + '/timeout',
+    timeout: 100
+  }
+
+  request(shouldTimeout, function(err, res, body) {
+    checkErrCode(t, err)
+    t.ok(err.host === 'localhost', 'Read Timeout Error should set \'host\' property')
+    t.ok(err.port === String(s.port), 'Read Timeout Error should set \'port\' property')
```

Comment:
use the actual value of s.port here (`'61160'`), similar to how you did on L76. That will make the test more accurate and solve your casting problem.